index.html

```diff
@@ -67,121 +67,84 @@ <h1 style="margin-top: .15em;">
-    </div>
-
-    <div class="col-md-3">&nbsp;9:50am</div>
-    <div class="col-md-9">
-      <b>EVA: An End-to-End Exploratory Video Analytics System</b><br/>
-      <i>Gaurav Tarlok Kakkar (Georgia Institute of Technology), Jiashen Cao (Georgia Tech), Pramod Chunduri (Georgia Institute of Technology), Zhuangdi Xu (Georgia Tech), Suryatej Reddy Vyalla (Georgia Institute of Technology), Anirudh Prabakaran (Georgia Institute of Technology), Jaeho Bang (Georgia Institute of Technology), Kaushik Ravichandran (Georgia Institute of Technology), Ishwarya Sivakumar (Georgia Institute of Technology), Aryan Rajoria (Georgia Institute of Technology), Ashmita Raju (Georgia Institute of Technology), Tushar Aggarwal (Georgia Institute of Technology), Shashank Suman (Georgia Institute of Technology), Myna Prasanna Kalluraya (Georgia Institute of Technology), Subrata Mitra (Adobe Research), Ali Payani (Cisco Systems Inc.), Yao Lu (Microsoft Research), Umakishore Ramachandran (Georgia Institute of Technology), Joy Arulraj (Georgia Tech)</i>
-    </div>
-
-    <div class="col-md-3">&nbsp;10:10am</div>
-    <div class="col-md-9">
-      <b>When Can we Ignore Missing Data in Model Training</b><br/>
-      <i>Cheng Zhen (Oregon State University), Amandeep Sing Chabada (Oregon State University), Arash Termehchy (Oregon State University)</i>
-    </div><br/>  
-    <br/><br/>&nbsp;<br/>
-
-    <div class="col-md-3"><b>&nbsp;10:30am - 11:00am</b></div><div class="col-md-9"><b>Break</b></div>
-
-    <br/>
-    <div class="col-md-12"></div><br/>
-    <div class="col-md-3"><b>&nbsp;11:00am - 12:20am</b></div><div class="col-md-9"><b>Session 2</b> - ML Optimization (Chair: Madelon Hulsebos)</div><br/>  
-    <div class="col-md-12"></div>
-
-    <div class="col-md-3">&nbsp;11:00am</div>
-    <div class="col-md-9">
-      <b>Using Pipeline Performance Prediction to Accelerate AutoML Systems</b><br/>
-      <i>Haoxiang Zhang (New York University), Roque Enrique López Condori (New York University), Aécio Santos (New York University), Jorge H Piazentin Ono (NYU), Aline Bessa (New York University), Juliana Freire (New York University)</i>
-    </div><br/>
-
-    <div class="col-md-3">&nbsp;11:20am</div>
-    <div class="col-md-9">
-      <b>P2D: A Transpiler Framework for Optimizing Data Science Pipelines</b><br/>
-      <i>Yordan Grigorov (Technische Universität Berlin), Haralampos Gavriilidis (Technische Universität Berlin), Sergey Redyuk (TU Berlin), Kaustubh Beedkar (IIT Delhi), Volker Markl (Technische Universität Berlin)</i>
-    </div><br/>  
-
-    <div class="col-md-3">&nbsp;11:40am</div>
-    <div class="col-md-9">
-      <b>Teaching Blue Elephants the Maths for Machine Learning</b><br/>
-      <i>Clemens Ruck (TUM); Maximilian E Schüle (University of Bamberg)*	1</i>
+    
+    <!-- Keynote -->
+    <div class="col-md-3"><b>9:00am - 10:00am</b></div>

```

Comment:
Opening notes could also start 8:50 if we want to have the keynote 60 minutes, although I think 50 minutes incl Q&A should be fine too.

Reply:
I thought about this, but 8:50 feels a bit early, especially since we are going all the way until 5pm